<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите имя компьютера, к которумо желаете подключиться:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editHostName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DESKTOP-P3MQMQK\MSSQLSERVER01</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите имя пользователя:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editUserName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HIkers01</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>2</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT!!!!!!</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="infoDB">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Info:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите имя базы данных:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDataBaseName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>winNT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите пароль:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>deus2508</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Отключиться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearInputCommand">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputInfoDB">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести информацию о базе данных</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendReguest">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>510</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить запрос</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radDeleteUsr1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить пользователя</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radDeleteGroup2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить группу</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radDeleteDomain3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить домен</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radEnumStation4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести станции данного домена</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radInfUsr5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>601</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Информация о пользователях в группе</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radDeleteGrNoDm6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>411</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить все группы, не входящие ни в один из доменов</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radDelUsrNoGpDm7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>581</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить пользователей, не входящих ни в одну группу и ни в один домен</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод входных данных:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="argData">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="outPutResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>751</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="argData2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="sumUser">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>540</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество строк в выборке запроса:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод дополнительных входных данных:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню таблиц</string>
    </property>
    <addaction name="actionUsers"/>
    <addaction name="actionStation"/>
    <addaction name="actionStation2"/>
    <addaction name="actionGr1"/>
    <addaction name="actionGr2"/>
    <addaction name="actionDomain"/>
    <addaction name="actionGrDo"/>
    <addaction name="actionDoSt"/>
    <addaction name="actionUsDo"/>
    <addaction name="actionUsGr"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionStation">
   <property name="text">
    <string>Station1</string>
   </property>
  </action>
  <action name="actionStation2">
   <property name="text">
    <string>Station2</string>
   </property>
  </action>
  <action name="actionGr1">
   <property name="text">
    <string>Gr1</string>
   </property>
  </action>
  <action name="actionGr2">
   <property name="text">
    <string>Gr2</string>
   </property>
  </action>
  <action name="actionDomain">
   <property name="text">
    <string>Domain</string>
   </property>
  </action>
  <action name="actionGrDo">
   <property name="text">
    <string>GrDo</string>
   </property>
  </action>
  <action name="actionDoSt">
   <property name="text">
    <string>DoSt</string>
   </property>
  </action>
  <action name="actionUsDo">
   <property name="text">
    <string>UsDo</string>
   </property>
  </action>
  <action name="actionUsGr">
   <property name="text">
    <string>UsGr</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
